Civi/Core/DAO/Event/PostUpdate.php

```diff
@@ -22,11 +22,18 @@ class PostUpdate extends \Symfony\Component\EventDispatcher\Event {
    */
   public $object;
 
+  /**
+   * @var mixed
+   */
+  public $result;
```

Comment:
Is there really no consistent data type for $result

Reply:
@eileenmcnaughton Thanks for highlighting the docblocks as being in need of some TLC. I think that your observations apply more generally to this class - and most likely to other classes too. I'd be happy to tackle those in a separate PR.